system-tests/playwright/playwright.todolistapp/model/Base.page.js

```diff
@@ -0,0 +1,10 @@
+class BasePage{
+    constructor(page){
+        this.page = page;
+    }
+
+    async navigate(path){
+        await this.page.goto(`https://single-page-application-fg5blhx72q-ey.a.run.app/${path}`)
```

Comment:
```suggestion
        await this.page.goto(`${baseUrl}/${path}`)
```
can you pass in the url somehow? so we can easily switch between "run on localhost" and "run on production"?